### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by binhle410**
I am targetting this branch, because FOSUserBundle has removed Document folder and replaced it with Doctrine. So SonataUser's counterpart needs updateing.
## Changelog

``` markdown
### Changed
- Document to Doctrine in GroupManager and UserManager
```
## To do

- [ ] Update the tests

### CODE REVIEW

File: Document/GroupManager.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 namespace Sonata\UserBundle\Document;
 
-use FOS\UserBundle\Document\GroupManager as BaseGroupManager;
+use FOS\UserBundle\Doctrine\GroupManager as BaseGroupManager;
```
Comment (greg0ire) [inline Document/GroupManager.php]: What's wrong with [this class](https://github.com/sonata-project/SonataUserBundle/blob/3.x/Document/GroupManager.php) ?

**Comment by greg0ire** [comment]
Can you find since which version it is like this?

**Comment by binhle410** [comment]
It s been removed since FOSUserBundle v2.0.0-alpha2.
There are some more bugs, let me find out what they are or maybe we can wait for the stable release of v2 and fix them all together.

**Comment by greg0ire** [comment]
I think 2.0 might get a release very soon, maybe even this weekend.

**Comment by jordisala1991** [comment]
Lets check what does travis says now

**Comment by jordisala1991** [comment]
Thanks @binhle410

**Pull request merged**